diff --git a/modules/gradio_hijack.py b/modules/gradio_hijack.py
index 817da40c..e871531b 100644
--- a/modules/gradio_hijack.py
+++ b/modules/gradio_hijack.py
@@ -74,7 +74,9 @@ def repair(grclass):
     grclass.update = gr.update
 
 
-for component in set(gr.components.__all__ + gr.layouts.__all__):
+components_list = gr.components.__all__ if hasattr(gr.components, '__all__') else []
+layouts_list = gr.layouts.__all__ if hasattr(gr.layouts, '__all__') else []
+for component in set(components_list + layouts_list):
     repair(getattr(gr, component, None))
 
 
diff --git a/modules/ui.py b/modules/ui.py
index 12f43768..ba979819 100644
--- a/modules/ui.py
+++ b/modules/ui.py
@@ -99,7 +99,6 @@ if not shared.args.old_colors:
         checkbox_label_background_fill_hover_dark='var(--selected-item-color-dark, #282930)',
         table_even_background_fill_dark='var(--darker-gray, #1C1C1D)',
         table_odd_background_fill_dark='var(--selected-item-color-dark, #282930)',
-        code_background_fill_dark='var(--darker-gray, #1C1C1D)',
 
         # Shadows and Radius
         checkbox_label_shadow='none',
